### PR TITLE
Adds RN for egress IP additional network interface support

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-8296

Link to docs preview:
https://66605--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-egress-ips-additional-networks

QE review:
No need.

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -468,6 +468,13 @@ Certain HTTP request and response headers can now be set or deleted either globa
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-set-or-delete-http-headers_configuring-ingress[Setting or deleting HTTP request and response headers in an Ingress Controller] and xref:../networking/routes/route-configuration.adoc#nw-route-set-or-delete-http-headers_route-configuration[Setting or deleting HTTP request and response headers in a route].
 
+[id="ocp-4-14-egress-ips-additional-networks"]
+==== Support for egress IPs on additional network interfaces (Technology Preview)
+
+Support for egress IPs on additional network interfaces is now available as a Technology Preview feature. This feature provides {ocp} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements.
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply release_notes/ocp-4-14-release-notes.adoc:474]: @mburke5678 Sorry. This is the abbreviation i use for Quay docs. It should be {product-title} here. Doh.

**Comment by stevsmit** [comment]
Requesting another round of peer review due to updates.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -468,6 +468,13 @@ Certain HTTP request and response headers can now be set or deleted either globa
...
 
+[id="ocp-4-14-egress-ips-additional-networks"]
+==== Support for egress IPs on additional network interfaces (Technology Preview)
+
+Support for egress IPs on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements. 
+
+For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc#nw-egress-ips-multi-nic-considerations_configuring-egress-ips-ovn[Consideration for using an egress IP on additional network interfaces].
```
Comment (kcarmichael08) [inline release_notes/ocp-4-14-release-notes.adoc:476]:
```suggestion
For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc#nw-egress-ips-multi-nic-considerations_configuring-egress-ips-ovn[Considerations for using an egress IP on additional network interfaces].
```

**Review by kcarmichael08 — COMMENTED** [review]
One small issue; otherwise LGTM!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (stevsmit) [reply release_notes/ocp-4-14-release-notes.adoc]: ty

**Pull request merged**